### PR TITLE
New package: Nheko-Reborn.nheko version 0.11.3

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132083)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is bot account and belongs to CoolPlayLin. Any issue should call him up instead of assigning to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let CoolPlayLin know).

Have a good day!

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification. It has the write permission of this account's fork so that it can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
image

File: manifests/n/Nheko-Reborn/nheko/0.11.3/Nheko-Reborn.nheko.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PackageVersion: 0.11.3
+UpgradeBehavior: install
+ReleaseDate: 2023-02-23
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://github.com/Nheko-Reborn/nheko/releases/download/v0.11.3/nheko-v0.11.3-installer.exe
+  InstallerSha256: 693C3E977C67723BD9CCC8B41812AB896899FEF64C0ADA1C0A254F2FE818E0A1
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/n/Nheko-Reborn/nheko/0.11.3/Nheko-Reborn.nheko.installer.yaml:14]:
```suggestion
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
  InstallerType: exe
  Scope: user
  InstallerUrl: https://github.com/Nheko-Reborn/nheko/releases/download/v0.11.3/nheko-v0.11.3-installer.exe
  InstallerSha256: 693C3E977C67723BD9CCC8B41812AB896899FEF64C0ADA1C0A254F2FE818E0A1
- Architecture: x64
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by Exorcism0666** [comment]
@stephengillie Idk what's wrong
image

**Comment by stephengillie** [comment]
> @ stephengillie Idk what's wrong

I'm also unclear.

**Comment by stephengillie** [comment]
> Possible duplicate package entry.
> 
> Similar installer SHA256 hash found in manifest:
> 
> * `InstallerSha256: 693c3e977c67723bd9ccc8b41812ab896899fef64c0ada1c0a254f2fe818e0a1`
>   
>   * manifests\n\NhekoReborn\Nheko\0.11.3

This package should be renamed to `NhekoReborn.Nheko` to match the existing packages. This version was [already added](https://github.com/microsoft/winget-pkgs/pull/99924), but it seems that [most versions were removed for automatically accepting licenses](https://github.com/microsoft/winget-pkgs/pull/130929), which violates most licenses.  

This manifest also accepts licenses, as the `--accept-licenses` in line 10:

```
  Silent: --accept-licenses --confirm-command install
```

**Pull request closed without merging**